pkg/controller/operators/catalog/operator.go

```diff
@@ -1100,6 +1100,18 @@ func (o *Operator) unpackBundles(plan *v1alpha1.InstallPlan) (bool, *v1alpha1.In
 		// Add steps and remove resolved bundle lookup
 		out.Status.Plan = append(out.Status.Plan, steps...)
 		out.Status.BundleLookups = append(out.Status.BundleLookups[:i], out.Status.BundleLookups[i+1:]...)
+		// add csv name to the list
```

Comment:
We should know these names before we unpack the bundles - they're returned as a part of the resolved set of operators from the resolver